### ISSUE TITLE
loss function

### ISSUE THREAD

**Issue by xuebaliang** [AltschulerWu-Lab/scScope#7]
hello, I recently read your paper and it is a perfect work. But I have some question about the loss function. Based on your loss function, I want to know how the parameters of  imputer layer update .
It is likely not to use the Wu and Wv in imputer.

**Comment by fbaothu** [comment]
Thanks for your interest. All parameters are updated through back propogation. Our recurrent model can be unfolded into a traditional structure as shown in Supplementary Figure 10. Therefore, all parameters in all layers can be learned in the same manner as traditional neural networks. Hope this could help.

Feng

**Comment by fbao-fudan** [comment]
> hello, I recently read your paper and it is a perfect work. But I have some question about the loss function. Based on your loss function, I want to know how the parameters of imputer layer update .
> It is likely not to use the Wu and Wv in imputer.

Thanks for your interest. All parameters are updated through back propogation. Our recurrent model can be unfolded into a traditional structure as shown in Supplementary Figure 10. Therefore, all parameters in all layers can be learned in the same manner as traditional neural networks. Hope this could help.

Feng

**Comment by xuebaliang** [comment]
ohoh, thanks。I have another question, what is the dimension default setting of the encoder layers and decoder layers and the latent layer？

**Comment by fbao-fudan** [comment]
> ohoh, thanks。I have another question, what is the dimension default setting of the encoder layers and decoder layers and the latent layer？

Please refer to the Methods part for details.
